src/StaticFAQ/ArticleFeedback/ScreenThankyou.js

```diff
@@ -37,12 +37,14 @@ const ScreenThankyou = (props: Props) => {
   return (
     <Box border="none" borderRadius="4px" backgroundColor="#f5f7f9">
       <div className="thank-you">
-        <div className="close-icon">
-          <Close
-            fill="#bac7d5"
-            height="12"
-            onClick={() => props.changeScreen(screensList.INITIAL)}
-          />
+        <div
+          className="close-icon"
+          onClick={() => props.changeScreen(screensList.INITIAL)}
+          onKeyUp={() => props.changeScreen(screensList.INITIAL)}
```

Comment:
Maybe we can just `onKeyUp={null}`? This would clarify that this listener is added just in order to pass lint rules and serves no actual purpose.